### ISSUE TITLE
[Enhancement] Add a github action to compile the LaTeX and push to a branch

### ISSUE THREAD

**Issue by duetosymmetry** [arkabokshi/asymptotics-perturbation-methods#1]
Can create a workflow using [xu-cheng/latex-action@v2](https://github.com/xu-cheng/latex-action) and [ad-m/github-push-action@master](https://github.com/ad-m/github-push-action). An [example workflow can be found here](https://github.com/duetosymmetry/LaTeX-template/blob/main/.github/workflows/pdflatex.yml).

**Comment by arkabokshi** [comment]
Thanks for the suggestion @duetosymmetry. For now I have added the `.pdf` document which was missing. I'll familiarize myself with the example workflow as soon as possible. What I had in mind was

1. Use the GitFlow workflow to make changes to the `Develop` branch
2. Once the `Develop` features have been merged into the `Main`, just manually generate the updated `.pdf` 

Suppose you are recommending that the pdf is autogenerated from the `Main` branch and autopushed to a `Release` branch?
